Brisk/Extensions/NSTextView+Extension.swift

```diff
@@ -2,12 +2,12 @@ import AppKit
 
 extension NSTextView {
     var stringValue: String {
-        return self.string ?? ""
+        return self.string
```

Comment:
Nice. I wonder if we can remove this property now

Reply:
👍